cookbooks/cdo-apps/recipes/default.rb

```diff
@@ -31,4 +31,24 @@
   user node[:current_user]
   group node[:current_user]
   action :nothing
+  notifies :run, 'execute[precompile-assets]', :immediately
+end
+
+execute "precompile-assets" do
+  command "rake assets:precompile"
```

Comment:
Yes, good catch.  I submit a PR that changes that.

On Mon, Oct 19, 2015 at 9:04 AM, Will Jordan <notifications@github.com>
wrote:

> In cookbooks/cdo-apps/recipes/default.rb
> <https://github.com/code-dot-org/code-dot-org/pull/4495#discussion_r42390309>
> :
>
> > @@ -31,4 +31,24 @@
> >    user node[:current_user]
> >    group node[:current_user]
> >    action :nothing
> > +  notifies :run, 'execute[precompile-assets]', :immediately
> > +end
> > +
> > +execute "precompile-assets" do
> > +  command "rake assets:precompile"
>
> @philbogle <https://github.com/philbogle> should this command be
> prepended with bundle exec?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/4495/files#r42390309>.
>